devices/MiHumidifierCAL.js

```diff
@@ -0,0 +1,49 @@
+const MiHumidifierV1 = require('./MiHumidifierV1');
+
+module.exports = class extends MiHumidifierV1 {
+    constructor() {
+        super();
+        this.version = "cal";
```

Comment:
Correct version is "ca1":

```suggestion
        this.version = "ca1";
```

Also the name of file in this case should be "MiHumidifierCA1"

Reply:
Fixed